sdcm/cluster.py

```diff
@@ -1819,6 +1819,17 @@ def wait_for_init(self, node_list=None, verbose=False, timeout=None):
         self.get_scylla_version()
         self.set_traffic_control()
 
+    def restart_scylla(self, nodes=None):
+        if nodes:
+            nodes_to_restart = nodes
+        else:
+            nodes_to_restart = self.nodes
+        self.log.info("Going to restart Scylla on %s" % nodes_to_restart)
```

Comment:
self.nodes - it's object, isn't? Do you want to print the node's name, not object, right?

Reply:
I want to print the list of all nodes that I am going to restart

Reply:
I mean, it's not node's names will be printed, but whole object. Or am I wrong?

Reply:
OK, it can be better to print node names only instead the whole objects str representations...